server/src/main/java/org/apache/iotdb/db/mpp/execution/schedule/DriverTaskThread.java

```diff
@@ -44,17 +45,21 @@ public class DriverTaskThread extends AbstractDriverThread {
   private static final Executor listeningExecutor =
       IoTDBThreadPoolFactory.newCachedThreadPool("scheduler-notification");
 
+  private final Ticker ticker;
+
   public DriverTaskThread(
       String workerId,
       ThreadGroup tg,
       IndexedBlockingQueue<DriverTask> queue,
       ITaskScheduler scheduler,
       ThreadProducer producer) {
     super(workerId, tg, queue, scheduler, producer);
+    this.ticker = Ticker.systemTicker();
```

Comment:
I don't know the real impaction between your usage and the one of official's suggestion.
![image](https://user-images.githubusercontent.com/3821212/211516243-73c1861b-d5bc-4ec8-a0fb-14de0629a98c.png)